behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx

```diff
@@ -7,18 +7,41 @@
 import React from 'react';
 import WhiteBackground from "../../whiteBackground/WhiteBackground.jsx";
 import PageLayout from "../../../enums/PageLayout";
+import MessagesPreview from './MessagesPreview.jsx';
 
 /**
  * Messages page with all users they've chatted with
  */
+
+// Dummy data
+const messagePreviews = [
+    {
+        userName: 'titan1',
+        name: 'Annie',
+        recentMessage: 'Hi, I was wondering if you were the amazing artist that he recommended?',
+        image: '',
```

Comment:
From the way the database schema is currently written you will be getting a photoid with the message object which you might have to fetch from the photos db: https://lucid.app/lucidchart/bf2238cb-8db1-4a45-a72d-b6007c65c46e/edit?viewport_loc=-3967%2C-1545%2C10533%2C3907%2C0_0&invitationId=inv_99b49be3-2c60-4b42-a5c7-16051f9d3b7e

Reply:
Let us know if you have a better way you'd like to do it